build.gradle

```diff
@@ -229,11 +229,15 @@ allprojects {
                     force "com.google.errorprone:error_prone_annotations:${googleErrorProneAnnotationsVersion}"
                     // different google APIs bring in different versions; force the latest
                     force "com.google.http-client:google-http-client-gson:${googleHttpClientGsonVersion}"
+                    // Force patched version of GPRC, dependency of
```

Comment:
"dependency of" what?

Reply:
I was trying to build suspense but went ahead and completed the thought